website/docs/guides/best-practices/materializations/materializations-guide-4-incremental-models.md

```diff
@@ -114,7 +114,7 @@ So we’re going to use an **if statement** to apply our cutoff filter **only wh
 
 Thankfully, we don’t have to dig into the guts of dbt to sort out each of these conditions individually.
 
-- ⚙️  dbt provides us with a **macro `[is_incremental](https://docs.getdbt.com/docs/build/incremental-models#understanding-the-is_incremental-macro)`** that checks all of these conditions for this exact use case.
+- ⚙️  dbt provides us with a **macro [`is_incremental`](https://docs.getdbt.com/docs/build/incremental-models#understanding-the-is_incremental-macro)** that checks all of these conditions for this exact use case.
```

Comment:
We should use relative links in the docs:
```suggestion
- ⚙️  dbt provides us with a **macro [`is_incremental`](/docs/build/incremental-models#understanding-the-is_incremental-macro)** that checks all of these conditions for this exact use case.
```

Reply:
will scan for these across the guide!